pkg/osquery/tables/apple_silicon_security_policy/table_test.go

```diff
@@ -0,0 +1,194 @@
+package apple_silicon_security_policy
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_ParseBootPoliciesOutputUnexpected(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name           string
+		status         string
+		expectedResult map[string]string
+		queryClause    []string
+	}{
+		{
+			name:   "bad output",
+			status: "\n\n\n\n",
+		},
+		{
+			name:   "no volume group",
+			status: "This utility is not meant for normal users or even sysadmins.",
+		},
+		{
+			name: "only volume group",
+			status: `Use at your own risk!
+			Local policy for volume group 5D0D176D-E8CC-**REDACTED**:`,
+		},
+		{
+			name: "no column data",
+			status: `Use at your own risk!
+			Local policy for volume group 5D0D176D-E8CC-**REDACTED**:
+			OS Policy:
+			Environment:`,
+		},
+	}
+
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+
+			o := parseBootPoliciesOutput(bytes.NewReader([]byte(tt.status)))
+			require.Empty(t, o)
+		})
+	}
+}
+
+const oneBootVol = `This utility is not meant for normal users or even sysadmins.
```

Comment:
Consider dropping these into embedded files. (We often use `test-data/...` for such)

Reply:
Good call, much better than these ugly literals